docs/source/pages/documentation.md

```diff
@@ -90,10 +90,9 @@ ssh \
 --central_host_id ssh.swc.ucl.ac.uk \
 --central_host_username username \
 --transfer_verbosity v \
---use-ephys --use-behav --use-histology --overwrite_old_files
+--use-ephys --use-behav --use-anat --overwrite_old_files
```

Comment:
These flags need to be removed, right? My understanding was that they are no longer configured, and have to be supplied each time.